src/main.rs

```diff
@@ -2,6 +2,14 @@
 #![plugin(rocket_codegen)]
 // #![cfg_attr(feature = "cargo-clippy", allow(new_ret_no_self))]
 
+//! Rustbox: A Rust based implementation of the Firefox Accounts Sync Storage service
```

Comment:
nit and probably just my preference: this comment can go at the very top, #![feature/plugin works fine afterwards